index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" >
+</head>
+<body>
+    <main id="main">
+  
+        <h1 id="title">Dr. Norman Borlaug</h1>
+
+         <div id="img-div">
+
+            <img id="image" src="https://images.pexels.com/photos/15746942/pexels-photo-15746942.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=1" alt="">
+              <div id="img-caption">
+                THis is a nice image caption
+              </div>
+
+         </div>
```

Comment:
Please always consider adding  the appropriate tags for each element (e.g., links, titles, etc.) and use HTML5 semantic tags (e.g., header, nav) over divs. Here, the contents of div "img-div" could be wrapped in <header> tag 
💯